test/eigen.jl

```diff
@@ -213,8 +213,8 @@ using StaticArrays, Test, LinearAlgebra
         @test eigvals(m) ≈ sort(m_d)
         @test eigvals(Hermitian(m)) ≈ sort(m_d)
 
-        # not Hermitian
-        @test_throws Exception eigen(@SMatrix randn(4,4))
+        # not Hermitian - we can handle these now
+        # @test_throws Exception eigen(@SMatrix randn(4,4))
```

Comment:
Feel free to delete if it's obsolete! Though I guess we should have a test for `eigen(Hermitan(SMatrix(...))))` doing "something sensible"

Reply:
Done, now testing the handling of rotation matrices (which are the first type of non-hermitian matrices with testable properties I came up with) and type stability on hermitian matrices.